@xen-orchestra/web/src/components/vm/network/VmVifSidePanel.vue

```diff
@@ -41,136 +41,69 @@
+            <!-- TODO Remove the span when the link works and the icon is fixed -->
+            <!--
               <UiComplexIcon size="medium">
                 <VtsIcon :icon="faNetworkWired" accent="current" />
                 <VtsIcon accent="success" :icon="faCircle" :overlay-icon="faCheck" />
               </UiComplexIcon>
               <a href="">{{ networkNameLabel }}</a>
               -->
-              <span v-tooltip class="text-ellipsis value">{{ network?.name_label }}</span>
-            </template>
-            <template v-if="network?.name_label" #addons>
-              <VtsCopyButton :value="network.name_label" />
-            </template>
-          </VtsCardRowKeyValue>
+          </UiLabelValue>
           <!-- DEVICE -->
-          <VtsCardRowKeyValue>
-            <template #key>
-              {{ t('device') }}
-            </template>
-            <template #value>
-              {{ t('vif-device', { device: vif.device }) }}
-            </template>
-            <template #addons>
-              <VtsCopyButton :value="vif.device" />
-            </template>
-          </VtsCardRowKeyValue>
+          <UiLabelValue
+            :label="t('device')"
+            :value="t('vif-device', { device: vif.device })"
+            :copy-value="vif.device"
+            ellipsis
+          />
           <!-- VIF STATUS -->
-          <VtsCardRowKeyValue>
-            <template #key>
-              {{ t('vif-status') }}
-            </template>
+          <UiLabelValue :label="t('vif-status')" ellipsis>
             <template #value>
               <VtsStatus :status />
             </template>
-          </VtsCardRowKeyValue>
+          </UiLabelValue>
           <!-- MTU -->
-          <VtsCardRowKeyValue>
-            <template #key>
-              {{ t('mtu') }}
-            </template>
-            <template #value>
-              {{ vif.MTU }}
-            </template>
-            <template #addons>
-              <VtsCopyButton :value="String(vif.MTU)" />
-            </template>
-          </VtsCardRowKeyValue>
+          <UiLabelValue :label="t('mtu')" :value="String(vif.MTU)" :copy-value="String(vif.MTU)" ellipsis />
           <!-- LOCKING MODE -->
-          <VtsCardRowKeyValue>
-            <template #key>
-              {{ t('locking-mode') }}
-            </template>
-            <template #value>
-              {{ vif.lockingMode }}
-            </template>
-          </VtsCardRowKeyValue>
+          <UiLabelValue :label="t('locking-mode')" :value="vif.lockingMode" ellipsis />
           <!-- TX CHECK SUMMING -->
-          <VtsCardRowKeyValue>
-            <template #key>
-              {{ t('check-summing') }}
-            </template>
-            <template #value>
-              {{ vif.txChecksumming }}
-            </template>
-          </VtsCardRowKeyValue>
+          <UiLabelValue :label="t('check-summing')" :value="String(vif.txChecksumming)" ellipsis />
```

Comment:
```suggestion
          <UiLabelValue :label="t('check-summing')" :value="String(vif.txChecksumming)" />
```
I don't think the ellipsis is necessary here, do you?